modules/mod_video/support/z_video_convert.erl

```diff
@@ -166,6 +168,7 @@ video_convert(QueuePath, Mime, Context) ->
         " -preset medium "
         " -metadata:s:v:0 rotate=0 ").
 
+-spec video_convert_1(file:filename(), integer(), any(), #context{}) -> {ok, file:filename()}.
```

Comment:
The spec in line 171 uses a record, please define a type for the record and use that instead.